app/src/main/java/com/imashnake/animite/features/home/HomeScreen.kt

```diff
@@ -83,10 +92,13 @@ fun HomeScreen(
     val homeMediaType = rememberSaveable { mutableStateOf(MediaType.ANIME) }
     viewModel.setMediaType(homeMediaType.value)
 
-    val trendingList by viewModel.trendingMedia.collectAsState()
-    val popularList by viewModel.popularMediaThisSeason.collectAsState()
-    val upcomingList by viewModel.upcomingMediaNextSeason.collectAsState()
-    val allTimePopularList by viewModel.allTimePopular.collectAsState()
+    val trendingList by viewModel.trendingMedia.data.collectAsState()
+    val popularList by viewModel.popularMediaThisSeason.data.collectAsState()
+    val upcomingList by viewModel.upcomingMediaNextSeason.data.collectAsState()
+    val allTimePopularList by viewModel.allTimePopular.data.collectAsState()
+
+    val isRefreshing by viewModel.isRefreshing.collectAsState()
+    val isLoading by viewModel.isLoading.collectAsState()
```

Comment:
Where does `isLoading` get used?
If the answer is "with `isRefreshing`", then maybe the ViewModel should be flattening these for us?